pkg/block/block_linux.go

```diff
@@ -271,7 +284,7 @@ func disks(ctx *context.Context, paths *linuxpath.Paths) []*Disk {
 		driveType, storageController := diskTypes(dname)
 		// TODO(jaypipes): Move this into diskTypes() once abstracting
 		// diskIsRotational for ease of unit testing
-		if !diskIsRotational(ctx, paths, dname) {
+		if driveType != DRIVE_TYPE_MAPPER && !diskIsRotational(ctx, paths, dname) {
```

Comment:
we will probably need to review this flow more deeply, but that's totally material for another PR. This change LGTM for this PR